zerver/models.py

```diff
@@ -1081,7 +1081,7 @@ class RealmEmoji(models.Model):
             # ending with one of the punctuation characters.
             RegexValidator(
                 regex=r"^[0-9a-z.\-_]+(?<![.\-_])$",
-                message=gettext_lazy("Invalid characters in emoji name"),
+                message=_("Invalid characters in emoji name"),
```

Comment:
Have you tested this? The intent of this logic was certainly that we'd only translate the string when the exception is actually thrown, not at process startups, so if this isn't actually working, that's a bug. You can test by setting your language to another one and then trying to create a custom emoji with the wrong name.

Reply:
(The issue is that because this is part of the class definition, like some of the globals we have elsewhere, a non-lazy evaluation is processed then, not when handling a request, like it would be if we were defining a validator class inside a function that's going to use it)

Reply:
This makes sense. I dug into `RegexValidator` and found that `message` is evaluated when the validator is called in case it subsequently generates a `ValidationError`. So we do want to preserve the laziness here. Probably need to fix this in `django-stubs`.